src/docs/testing/integration-tests/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Integration Testing
+description: Learn how to write integration tests
+---
+
+This page describes how to use the [integration_test][] package to run
+integration tests. Tests written using this package have the following
+properties:
+ 
+- Compatibility with the `flutter drive` command, for running tests on a
+physical device or emulator.
+- The ability to be run on [Firebase Test Lab], enabling automated testing on a
+variety of devices.
+- Compatibility with [flutter_test] APIs, enabling tests to be written in a
+similar style as [widget tests][]
+
+## Overview
+
+If you're developing a Flutter app, you are probably writing the code on a
+desktop computer, called the **host** machine, and running the app on a mobile
+device or browser, called the **target**.
+
+There are three types of tests that Flutter supports. A **unit** test verifies
+the behavior of a method or class. A **widget test** verifies the behavior of
+Flutter widgets without running the app itself. An **integration test** runs the
+full app on the target device and reports the results back to the host machine.
+
+The flutter_driver package runs integration tests written in Dart on a target
+device and reports the results to the host.
+
+The [integration_test][] package lets you run tests directly on the target
+device. This allows you to run integration tests on multiple Android or iOS
+devices using Firebase TestLab. These tests can also be run using
+flutter_driver.
+
+## Project Setup
+
+Add `integration_test`, `flutter_test`, and optionally `flutter_driver` to your
+pubspec.yaml file:
+
+```yaml
+integration_test: ^0.9.0
+flutter_test:
+  sdk: flutter
```

Comment:
Do you need a particular version of the sdk?

Reply:
Yes, but that's enforced by `integration_test`'s pubspec, not by the user: https://github.com/flutter/plugins/blob/master/packages/integration_test/pubspec.yaml#L7

Reply:
Actually, that's a Dart SDK constraint, so no it doesn't need a particular version I don't think. cc: @dnfield @nturgut

Reply:
This is pub's way of saying "just pull this from the local flutter SDK installation" rather than from pub.dev.